File: src/com/codingspezis/android/metalonly/player/plan/views/SectionView.java
package com.codingspezis.android.metalonly.player.plan.views;

import com.codingspezis.android.metalonly.player.*;
import com.codingspezis.android.metalonly.player.plan.*;
import com.googlecode.androidannotations.annotations.*;

import android.content.*;
import android.util.*;
import android.widget.*;

@EViewGroup(R.layout.view_plan_section)
public class SectionView extends LinearLayout {

	@ViewById
	TextView title;
	
	public SectionView(Context context, AttributeSet attrs) {
		super(context, attrs);
	}

	public void bind(SectionItem sectionItem ){
		String title2 = sectionItem.getTitle();
		title.setText(title2);
	}
}

File: app/src/main/kotlin/com/codingspezis/android/metalonly/player/plan/views/SectionView.kt
package com.codingspezis.android.metalonly.player.plan.views

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout
import android.widget.TextView

import com.codingspezis.android.metalonly.player.R
import com.codingspezis.android.metalonly.player.plan.PlanSectionItem

import org.androidannotations.annotations.EViewGroup
import org.androidannotations.annotations.ViewById

/**
 * A view to divide sections in plans.
 */
@EViewGroup(R.layout.view_plan_section)
open class SectionView(context: Context, attrs: AttributeSet?) : LinearLayout(context, attrs), CustomDataView<PlanSectionItem> {

    @JvmField
    @ViewById(android.R.id.title)
    protected var textTitle: TextView? = null

    override fun bind(t: PlanSectionItem) {
        textTitle?.text = t.title
    }
}
